nasa/gesdisc-meditor
# Feature/meditor 858 bulk UI

This PR has following changes from the document list (model search) page:
Add one or more fields to be updated in the selected documents 
List (or search for) fields in a model. These fields can have multiple levels, for example: "Project.Name"
Allow user to input a value to change the field to
Show success/failure responses

To Test:
visit localhost/meditor, navigate to model page and select documents to change the fields. there is bulk select button on top which opens bulk update model. edit fields and submit your changes. it should close the model and show success toast after submission with number of documents updated. check those documents to make sure field has been updated.
